rotkehlchen/api/rest.py

```diff
@@ -2921,8 +2927,17 @@ def get_asset_icon(
             )
             if response is not None:
                 return response
+            else:
+                return maybe_create_image_response(image_path=icon_path)
```

Comment:
What are you doing here? Did you not already just check for the same icon path above?

Reply:
here is creating a the response but is true that adds a check that was already done. I created a new function to avoid that. Another system call saved